lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -10,21 +10,22 @@
 from django.urls import reverse
 from opaque_keys.edx.keys import CourseKey
 
+import lms.djangoapps.course_blocks.api as course_blocks_api
 from lms.djangoapps.course_api.blocks.transformers.blocks_api import BlocksAPITransformer
+from lms.djangoapps.course_blocks.api import get_course_blocks
```

Comment:
I realize this isn't you, but this feels a little weird, right?
Should we get rid of the import of `get_course_blocks` and instead just all `course_blocks_api.get_course_blocks`?

Reply:
Yeah, consolidated both imports.